_layouts/simple.html

```diff
@@ -53,29 +53,27 @@ <h2 class="addresses-header">Our locations</h2>
     {% endif %}
     {% if page.title == "Privacy Policy" %}
     <div class="story-list">
-        <span><a href="#English">English</a> /</span>
-        <span><a href="#Swahili">Swahili</a> /</span> </br>
-        <span><a href="#French">French</a> /</span>
-        <!-- <span><a href="#Hindi">Hindi</a> /</span> -->
+      <span><a href="/privacy-policy-english/" {% if page.url == "/privacy-policy-english/" %}style="color: #1D214F;"{% endif %}>English /</a></span>
+      <span><a href="/privacy-policy-swahili/" {% if page.url == "/privacy-policy-swahili/" %}style="color: #1D214F;"{% endif %}>Swahili /</a></span> <br>
+      <span><a href="/privacy-policy-french/" {% if page.url == "/privacy-policy-french/" %}style="color: #1D214F;"{% endif %}>French /</a></span>
     </div>
     {% endif %}
     {% if page.title != "Contact" %}
     <div class="success-story-wrapper">
-      {% if page.title == "Privacy Policy" %}<h2 id="English">English</h2>{% endif %}
+      {% if page.title == "Privacy Policy" %}
+      <!-- <h2 id="English">English</h2> -->
+      {% endif %}
       {{page.Body | markdownify }}
 
       {% if page.title == "Privacy Policy" %}
-      <hr></hr>
-      <h2 id="Swahili">Swahili</h2>
         {{page['Privacy Policy - Swahili Description'] | markdownify }}
       {% endif %}
 
       {% if page.title == "Privacy Policy" %}
-      <hr></hr>
-      <h2 id="French">French</h2>
         {{page['Privacy Policy - French Description'] | markdownify }}
       {% endif %}
```

Comment:
Since the condition is the same `if page.title == "Privacy Policy"` this can probably be unified into a single if.

Reply:
will do